### PR TITLE
Fix code coverage.

### LINKED ISSUE #6356: Code coverage test is broken again
https://grpc-testing.appspot.com/job/gRPC_coverage/

### PR DESCRIPTION
**Pull request by adelez**
To fix #6356.

### CODE REVIEW

File: tools/run_tests/run_tests.py [hunk 1]
```diff
@@ -892,9 +892,9 @@ def git(cmd, cwd=cwd):
 
 language_make_options=[]
 if any(language.make_options() for language in languages):
-  if len(languages) != 1:
-    print 'languages with custom make options cannot be built simultaneously with other languages'
-    sys.exit(1)
+  if not 'gcov' in args.config and len(languages) != 1:
+      print 'languages with custom make options cannot be built simultaneously with other languages'
```
Comment (jtattermusch) [inline tools/run_tests/run_tests.py]: why the extra indent?

**Comment by adelez** [comment]
The failures in basic tests seem unrelated.

File: tools/run_tests/run_tests.py (hunk 1 above)
Comment (adelez) [inline tools/run_tests/run_tests.py]: Fixed.

**Comment by jtattermusch** [comment]
This hasn't been LGTMd yet. Please don't merge yourself. Instead, wait for LGTM and add "ready to merge" label once you believe it's ready.

**Comment by adelez** [comment]
Sorry. Per offline discussion, I thought this was like LGTM with some nit work. Won't happen again.

**Pull request merged**